modules/services/deeplink/src/main/kotlin/au/com/shiftyjelly/pocketcasts/deeplink/DeepLink.kt

```diff
@@ -244,6 +244,10 @@ data class SignInDeepLink(
     }
 }
 
+data class PocketCastsWebsiteReferralDeepLink(
+    val code: String,
+) : DeepLink
```

Comment:
Do you think this would should be `UriDeepLink`? This way it could be used for sharing. If we add there are tests like `SignInDeepLinkTest` to be added.